blockchain/state_transition.go

```diff
@@ -170,21 +170,30 @@ func (st *StateTransition) buyGas() error {
 	validatedSender := st.msg.ValidatedSender()
 	feeRatio, isRatioTx := st.msg.FeeRatio()
 	if isRatioTx {
-		feePayer, feeSender := types.CalcFeeWithRatio(feeRatio, mgval)
+		feePayerFee, senderFee := types.CalcFeeWithRatio(feeRatio, mgval)
 
-		if st.state.GetBalance(validatedFeePayer).Cmp(feePayer) < 0 {
+		if st.state.GetBalance(validatedFeePayer).Cmp(feePayerFee) < 0 {
+			logger.Debug("%w - feePayer: %v, feePayerBalance: %v, feePayerFee: %v, txHash: %v",
```

Comment:
`logger` does not support this style.
`logger.Debug("blah %v", feePayerFee)`

Please write this sytle.
`logger.Debug("blah ", "feePayerFee",  feePayerFee)`

Reply:
Oh, I forgot to change the log from string to key value pairs while changing it from an error to the log. Thanks for checking the issue!